packages/klighd-core/src/options/general-panel.tsx

```diff
@@ -72,62 +62,9 @@ export class GeneralPanel extends SidebarPanel {
         return "General";
     }
 
-    private assignQuickActions() {
-        this.quickActions = [
-            {
-                key: "center",
-                title: "Center diagram",
-                iconId: "maximize",
-                action: CenterAction.create([], { animate: true }),
-            },
-            {
-                key: "fit",
-                title: "Fit to screen",
-                iconId: "maximize-2",
-                action: this.renderOptionsRegistry.getValue(ResizeToFit) ? undefined : KlighdFitToScreenAction.create(true),
-                state: this.renderOptionsRegistry.getValue(ResizeToFit),
-                effect: () => {
-                        this.actionDispatcher.dispatch(SetRenderOptionAction.create(ResizeToFit.ID, !this.renderOptionsRegistry.getValue(ResizeToFit)));
-                        this.update()
-                }
-            },
-            {
-                key: "layout",
-                title: "Layout diagram",
-                iconId: "layout",
-                action: RefreshLayoutAction.create(),
-            },
-            {
-                key: "refresh",
-                title: "Refresh diagram",
-                iconId: "rotate-cw",
-                action: RefreshDiagramAction.create(),
-            },
-            {
-                key: "export",
-                title: "Export as SVG",
-                iconId: "save",
-                action: RequestExportSvgAction.create(),
-            },
-            {
-                key: "create-bookmark",
-                title: "Bookmark",
-                iconId: "bookmark",
-                action: CreateBookmarkAction.create()
-            },
-            {
-                key: "pin-sidebar",
-                title: this.renderOptionsRegistry.getValueOrDefault(PinSidebarOption) ? "Unpin Sidebar" : "Pin Sidebar",
-                iconId: this.renderOptionsRegistry.getValueOrDefault(PinSidebarOption) ? "lock" : "unlock",
-                action: SetRenderOptionAction.create(PinSidebarOption.ID, !this.renderOptionsRegistry.getValueOrDefault(PinSidebarOption)),
-                state: this.renderOptionsRegistry.getValue(PinSidebarOption)
-            },
-        ];
-    }
-
     update(): void {
-        this.assignQuickActions()
-        super.update()
+        super.assignQuickActions();
+        super.update();
```

Comment:
Usually we have coding guidelines enforced by a Plugin called "Prettier", which somehow is not enabled in this repository yet - I may have a look at that later. One of the rules is that we do not allow the usage of these optional semicolons, so you should not explicitly add them where there are none before.